operator/data/profiles/demo.yaml

```diff
@@ -19,6 +19,11 @@ spec:
           requests:
             cpu: 10m
             memory: 40Mi
+        service:
+          ports:
+            - port: 31400
```

Comment:
does this add, not replace? is there any way for a user to replace?

Reply:
oh yes this should be replacing not addition, just updated the change to reflect this